lca-cli/ibi-preparation/ibipreparation_test.go

```diff
@@ -2,11 +2,12 @@ package ibi_preparation
 
 import (
 	"fmt"
-	"github.com/openshift-kni/lifecycle-agent/api/ibiconfig"
 	"os"
 	"path"
 	"testing"
 
+	"github.com/openshift-kni/lifecycle-agent/api/ibiconfig"
```

Comment:
There seems to be an issue with this change in the CI, specifically I noticed the following [here](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift-kni_lifecycle-agent/677/pull-ci-openshift-kni-lifecycle-agent-main-ci-job/1854191739665911808):

```shell
go vet ./...
# github.com/openshift-kni/lifecycle-agent/lca-cli/ibi-preparation
vet: lca-cli/ibi-preparation/ibipreparation_test.go:17:2: ibiconfig redeclared in this block
```

I didn't look into the cause of this, but I think if we moved this line to line 16 in the same file (i.e. before the other lifecycle-agent import) , it should resolved the CI issue. @xphyr could you perhaps give it a try?